docs/modules/ROOT/pages/advanced-how-tos/accessing-private-repositories.adoc

```diff
@@ -0,0 +1,71 @@
+= Access Pipelines and Tasks in Private Repositories
+
+Integration Test Scenarios point to their corresponding pipeline with a link:https://tekton.dev/docs/pipelines/git-resolver/[Git Resolver]. The Git Resolver schema is inherited from Tekton and functions in the same way. Users should create an access token for their repository in link:https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens[Github] or link:https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html[Gitlab] then link:../how-tos/configuring/creating-secrets.adoc#creating-source-control-secrets[store that token in a secret] in their namespace.
```

Comment:
Should this go within the testing section? We should also add it to the navigation pane so that readers can discover the page.